### PR TITLE
use ansi-color-control-seq-regexp in newer ansi-color

### PR DESCRIPTION
**Pull request by nverno**

### CODE REVIEW

File: sbt-mode-hydra.el [hunk 1]
```diff
@@ -28,6 +28,10 @@
 (require 'sbt-mode-buffer)
 (require 'sbt-mode)
 
+(defvar ansi-color-regexp)
```
Comment (hvesalai) [inline sbt-mode-hydra.el]: Isn't this the wrong file for this change. Shouldn't it be in sbt-mode-comint?

Comment (nverno) [reply sbt-mode-hydra.el]: I'm not familiar with the project. I just tried to use it, but the mode wasn't working so I added the fix to the file that contained the offending variable. I believe this was the only file that used `ansi-color-regexp`, but `sbt-mode-comint` certainly sounds like a more appropriate place

Comment (nverno) [reply sbt-mode-hydra.el]: I could resubmit it to be defined in the other file if you like. As it is, on newer versions of emacs the sbt comint wont function correctly

Comment (hvesalai) [reply sbt-mode-hydra.el]: Please do. What do you mean won't function correctly?

Comment (nverno) [reply sbt-mode-hydra.el]: I mean that the process-filter throws a variable undefined error

**Comment by hvesalai** [comment]
@fommil will you review. This changes color handling.

**Pull request merged**